Examples/Algorithms/Digitization/include/ActsExamples/Digitization/Smearers.hpp

```diff
@@ -26,6 +26,11 @@ namespace ActsExamples::Digitization {
 /// Exact smearing of a single parameter.
 ///
 struct Exact {
+  double sigma;
+
+  /// Construct with a @param sigma standard deviation
+  Exact(double sigma_) : sigma{sigma_} {}
```

Comment:
```suggestion
  explicit Exact(double sigma_) : sigma{sigma_} {}
```

Maybe we should do this for the other ones as well.